devtools/common/src/main/resources/templates/gitignore.ftl

```diff
@@ -0,0 +1,21 @@
+# Eclipse
+.project
+.classpath
+.settings/
+bin/
+
+# IntelliJ
+.idea
+*.ipr
+*.iml
+*.iws
+
+# NetBeans
+nb-configuration.xml
+
+# Visual Studio Code
+.vscode
+
+# OSX
+.DS_Store
```

Comment:
If you can add the Vim temporary files
```
*.swp
*.swo
```
that would be nice.

And maybe the diff conflicts:
```
*.orig
*.rej
```
too.